apps/volontulo/templates/offers/offer_form.html

```diff
@@ -5,73 +5,105 @@
             </div>
-            <label class="col-xs-2 control-label" for="{{ offer_form.location.id_for_label }}">Lokalizacja</label>
-            <div class="col-xs-4">
-                <input type="text" value="{{ request.POST.location|default:offer.location }}" name="location" class="form-control" id="{{ offer_form.location.id_for_label }}" />
+            <div class="col-xs-6 {% if form.location.errors %}has-error{% endif %}">
+                <div class="col-xs-12">{{ form.location.errors }}</div>
+                <div class="col-xs-12 form-group form-group-sm">
+                    <div class="col-xs-5"><label class="control-label" for="{{ form.location.id_for_label }}">Lokalizacja</label></div>
+                    <div class="col-xs-7"><input type="text" value="{{ request.POST.location|default:offer.location }}" name="location" class="form-control" id="{{ form.location.id_for_label }}" /></div>
+                </div>
             </div>
         </div>
-        <div class="form-group form-group-sm">
-            <label class="col-xs-2 control-label" for="{{ offer_form.time_commitment.id_for_label }}">Wymagania czasowe</label>
-            <div class="col-xs-4">
-                <input type="text" value="{{ request.POST.time_commitment|default:offer.time_commitment }}" name="time_commitment" class="form-control" id="{{ offer_form.time_commitment.id_for_label }}" />
-            </div>
-            <label class="col-xs-2 control-label" for="{{ offer_form.time_period.id_for_label }}">Dzień rozpoczęcia</label>
-            <div class="col-xs-4">
-                <input type="text" value="{{ request.POST.time_period|default:offer.time_period }}" name="time_period" class="form-control" id="{{ offer_form.time_period.id_for_label }}" />
+
+        <div class="row show-grid">
+            <div class="col-xs-6 {% if form.time_commitment.errors %}has-error{% endif %}">
+                <div class="col-xs-12">{{ form.time_commitment.errors }}</div>
+                <div class="col-xs-12 form-group form-group-sm">
+                    <div class="col-xs-5"><label class="control-label" for="{{ form.time_commitment.id_for_label }}">Wymagania czasowe</label></div>
+                    <div class="col-xs-7"><input type="text" value="{{ request.POST.time_commitment|default:offer.time_commitment }}" name="time_commitment" class="form-control" id="{{ form.time_commitment.id_for_label }}" /></div>
+                </div>
             </div>
-        </div>
-        <div class="form-group form-group-sm">
-            <label class="col-xs-offset-6 col-xs-2 control-label" for="{{ offer_form.time_period_end.id_for_label }}">Dzień Zakończenia</label>
-            <div class="col-xs-4">
-                <input type="text" value="" name="time_period_end" class="form-control" id="{{ offer_form.time_period_end.id_for_label }}" />
+            <div class="col-xs-6">
+                <div class="col-xs-12 {% if form.started_at.errors %}has-error{% endif %}">
```

Comment:
this is wrong. `col-`s cannot be nested without using `row` class element
but looking at it i think there is some unnecessary wrapping. 
This can be possibly fixed in different task